### ISSUE TITLE
suggest pastebin if more than a few lines are to be sent

### ISSUE THREAD

**Issue by nirik** [hexchat/hexchat#1537]
If someone accidentially pastes a large amount of lines in the send queue, it can flood channels. 

Perhaps by default hexchat could say "hey, this is more than 5 lines are you sure you don't want to use a pastebin"? 

Or perhaps even some kind of autopastebin happens (if configured) and it sends the link to the pastebin automagically.

**Comment by Arnavion** [comment]
Sending to a pastebin site should be done by scripts, but yes HC can show a Yes/No popup asking for confirmation to send >5 lines.

**Comment by TingPing** [comment]
Polari does this but they have the benefit of gnome infrastructure hosting a pastebin service. We certainly don't want to deal with that nor rely on some third party. An unofficial script is certainly the way to go.

Would be a fairly simple script; One would want to just hook keypress, check for new lines, then prompt or  pastebin and replace the input with a url.

**Comment by Burrito-Bazooka** [comment]
Try this: /hexchat/hexchat-addons/tree/master/python/floodcontrol

**Comment by nirik** [comment]
Yeah, that looks pretty nice... but I was hoping for some built in way. The folks that need this the most are new users to irc who wouldn't know to install the plugin. ;(

**Comment by TingPing** [comment]
Here is a script that adds a `/paste` command and uses fedora's pastebin: https://gist.github.com/TingPing/5993eeb9019f8b0798ad270f365dd6a4
